lute/parse/registry.py

```diff
@@ -24,13 +25,21 @@ def init_parser_plugins():
     """
     Initialize parsers from plugins
     """
-    custom_parser_eps = entry_points().get("lute.plugin.parse", [])
+
+    if version_info.major == 3 and version_info.minor in (8, 9, 10, 11):
+        custom_parser_eps = entry_points().get('lute.plugin.parse')
+    elif version_info.major == 3 and version_info.minor == 12:
```

Comment:
Perhaps we should change this to `>= 12`.  Granted, we can't know what the API will be in version 3.13+, but maybe we can play fast and loose with this one, and just bump up the CI checks on GitHub as versions come out.